libs/dal/src/repositories/organization/organization.repository.ts

```diff
@@ -57,4 +57,23 @@ export class OrganizationRepository extends BaseRepository<OrganizationEntity> {
       }
     );
   }
+
+  async bulkUpdatePartnerConfiguration(userId: string, data: Record<string, string[]>, configurationId: string) {
+    const members = await this.memberRepository.findUserActiveMembers(userId);
+    const allOrgs = members.map((member) => member._organizationId);
+    const usedOrgIds = Object.keys(data);
+    const unusedOrgIds = allOrgs.filter((org) => !usedOrgIds.includes(org));
+    const bulkWriteOps = allOrgs.map((orgId) => {
+      return {
+        updateOne: {
+          filter: { _id: orgId, 'partnerConfigurations.configurationId': configurationId },
+          update: {
+            'partnerConfigurations.$.projectIds': unusedOrgIds.includes(orgId) ? [] : data[orgId],
```

Comment:
this ternary operation helps to update the organizations that were de-linked during the update process,
so setting the projectIds to `[]`, marks the org id as not being linked with any vercel project